tools/UpdatePaths/Scripts/72586_tramstation_asteroid_area.txt

```diff
@@ -0,0 +1 @@
+/area/lavaland/surface : /area/misc/asteroid{@OLD}
```

Comment:
either delete this file (someone might mistakenly run it and screw up lavaland or something on a downstream) or write a comment about it's use.

it's probably 500x better to delete it tbh since people mindlessly run updatepaths sometimes. i just suggested the updatepaths way of doing it to ensure you didn't accidentally generate any map key cruft

Reply:
Sure.